spark-commons/src/test/scala/za/co/absa/spark/commons/errorhandling/implementations/ErrorHandlingFilterRowsWithErrorsTest.scala

```diff
@@ -87,4 +89,35 @@ class ErrorHandlingFilterRowsWithErrorsTest extends AnyFunSuite with SparkTestBa
 
     assert(result == expected)
   }
+
+  test("errorColumnType should return a BooleanType") {
+    val errorColumn: ErrorColumn = ErrorHandlingFilterRowsWithErrors.putErrorToColumn(
+      "Test error 1", 1, "This is a test error", Some(errColName))
+
+    val testDf = emptyDf.withColumn(errColName, errorColumn.column)
+    val expectedResults = testDf.col(errColName).expr.dataType
+    val expval = testDf.schema.fields
+
+    val results = ErrorHandlingFilterRowsWithErrors.errorColumnType
+
+    assert(results.defaultSize == expval.length)
+    assert(results == expectedResults)
+  }
+
+  test("errorColumnAggregationType should return None since no column is added during the aggregation") {
+    val errorColumn: ErrorColumn = ErrorHandlingFilterRowsWithErrors.putErrorToColumn(
+      "1st error", 0, "This is an error", Some(errColName)
+    )
+
+    val testDf = emptyDf
+    val expVal = testDf.schema.fields.toList.headOption
+
+    val expectedAfterAgg = ErrorHandlingFilterRowsWithErrors.aggregateErrorColumns(testDf)(errorColumn)
```

Comment:
This is not 'expected' but 'actual', right? You are actually performing some computation. So something like `expectedAfterAgg` -> `actualDfAfterAgg`